src/AppServices/KafkaPageV3.tsx

```diff
@@ -0,0 +1,327 @@
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardHeaderMain,
+  CardTitle,
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Divider,
+  Flex,
+  FlexItem,
+  Grid,
+  Split,
+  SplitItem,
+  Stack,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+} from "@patternfly/react-core";
+import type { FunctionComponent } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { AppSpeedKeyArt, LogoAWS, LogoAzure, LogoGCP } from "../images";
+import {
+  KafkaInstanceCapacityTable,
+  MarketingPageHero,
+  MarketingPageSection,
+} from "./components";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons";
+import { ExternalLink } from "../shared";
+
+export const KafkaPageV3: FunctionComponent = () => {
+  const { t } = useTranslation(["kafkaoverview-v3"]);
+  return (
+    <>
+      <MarketingPageHero
+        title={t("heroTitle")}
+        tagLine={t("heroTagline")}
+        description={t("heroDescription")}
+        heroImage={AppSpeedKeyArt}
+        heroImageSize={478}
+        heroImageCanRepeat={false}
+        heroImagePositionY={-99}
+      />
+      <MarketingPageSection>
+        <Grid hasGutter lg={6}>
+          <Card
+            ouiaId="card-overview-purchase-now"
+            aria-label={t("kafkaoverview:purchaseCardTitle")}
+          >
+            <CardHeader>
+              <CardHeaderMain>
+                <CardTitle>
+                  <Title headingLevel="h2">
+                    <Trans
+                      ns={"kafkaoverview-v3"}
+                      i18nKey={"purchaseCardTitle"}
+                    >
+                      <span className="pf-u-color-200 pf-u-font-size-sm"></span>
+                    </Trans>
+                  </Title>
+                </CardTitle>
+              </CardHeaderMain>
+            </CardHeader>
+            <CardBody>{t("purchaseCardMainText")}</CardBody>
+            <CardFooter>
+              <Flex>
+                <FlexItem>
+                  <Button
```

Comment:
I think all these buttons could be converted to use the `ExternalLink` component.

Reply:
Yeah, I'll do that